### PR TITLE
Sign up implementation

### PR DESCRIPTION
**Pull request by erenGuden**
Add "/signup" page and logic implementation.
Closes issue #7.

### CODE REVIEW

File: src/components/Login.jsx [hunk 1]
```diff
@@ -23,12 +24,15 @@ const GridStyled = styled(Grid)(({ theme }) => ({
   },
 }));
 
+// /api/users?username=
```
Comment (adamkoksal) [inline src/components/Login.jsx:27]: remove this

File: src/components/Signup.jsx [hunk 2]
```diff
@@ -1,7 +1,83 @@
...
+    axios
+      .post(baseurl, {
+        username,
+        password,
+        address,
+        securityQuestion,
+        securityQuestionAnswer,
+      })
+      .then((response) => {
+        console.log(response);
```
Comment (adamkoksal) [inline src/components/Signup.jsx:52]: remove console log

File: src/components/Signup.jsx [hunk 3]
```diff
@@ -1,7 +1,83 @@
...
+        setError(true);
+        setError(error.response.data);
+      });
+  };
+
+  const onChangeUsername = (e) => {
+    e.preventDefault();
+
+    setUsername(e.target.value);
+    console.log(e.target.value);
```
Comment (adamkoksal) [inline src/components/Signup.jsx:68]: remove all logs

**Comment by erenGuden** [comment]
Fixed the code and removed. unnecessary  lines.

**Review by adamkoksal — APPROVED** [review]

**Pull request merged**